src/routes/consent.ts

```diff
@@ -10,7 +10,38 @@ import {
 } from "@ory/client"
 import { UserConsentCard } from "@ory/elements-markup"
 import bodyParser from "body-parser"
-import csrf from "csurf"
+import { doubleCsrf } from "csrf-csrf"
+import { Request, Response, NextFunction } from "express"
+
+// Sets up csrf protection
+const {
+  generateToken, // Use this in your routes to provide a CSRF hash + token cookie and token.
+  invalidCsrfTokenError,
+  doubleCsrfProtection, // This is the default CSRF protection middleware.
+} = doubleCsrf({
+  getSecret: () => "VERY_SECRET_VALUE", // A function that optionally takes the request and returns a secret
+  cookieName: "ax-x-csrf-token", // The name of the cookie to be used, recommend using Host prefix.
+  cookieOptions: {
+    sameSite: "lax", // Recommend you make this strict if posible
+    secure: true,
```

Comment:
in dev environments (non-https) this needs to be false. I guess we should add another env variable for this